src/test/java/com/wikia/webdriver/common/core/api/GraphApi.java

```diff
@@ -0,0 +1,127 @@
+ */
+public class GraphApi {
+
+    private static String ERROR_MESSAGE = "Problem with Graph API call used to create new facebook test user";
+    private static ArrayList<BasicNameValuePair> PARAMS;
+    private static String wikia_production_app_access_token = "112328095453510|uncl945c48bixtm47AWZM64doDQ";
+    private static String wikia_preview_app_access_token = "983287518357559|RE3IANMJOSmv04gSFO7hThMWVBI";
+
+    public HashMap<String, String> createFacebookTestUser(String app_id) {
+        try {
+            HttpResponse response = createTestUser(app_id);
+            InputStream body = response.getEntity().getContent();
+            String entity = EntityUtils.toString(response.getEntity());
+            HashMap<String, String> result =
+                new Gson().fromJson(entity, new TypeToken<HashMap<String, String>>(){}.getType());
+            return result;
+        }
+        catch (IOException e) {
+            PageObjectLogging.log("URI_SYNTAX EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+            throw new WebDriverException(ERROR_MESSAGE);
+        }
+        catch (URISyntaxException e) {
+            PageObjectLogging.log("URI_SYNTAX EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+            throw new WebDriverException(ERROR_MESSAGE);
+        }
+
+    }
+
+    public HashMap<String, String> deleteFacebookTestUser(String app_id, String user_id) {
+        try {
+            HttpResponse response = deleteTestUser(user_id);
+            InputStream body = response.getEntity().getContent();
+            String entity = EntityUtils.toString(response.getEntity());
+            HashMap<String, String> result =
+                new Gson().fromJson(entity, new TypeToken<HashMap<String, String>>(){}.getType());
+            return result;
+        }
+        catch (IOException e) {
+            PageObjectLogging.log("URI_SYNTAX EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+            throw new WebDriverException(ERROR_MESSAGE);
+        }
+        catch (URISyntaxException e) {
+            PageObjectLogging.log("URI_SYNTAX EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+            throw new WebDriverException(ERROR_MESSAGE);
+        }
+
+    }
+
+    private String getURLcreateUser(String app_id) {
+        String host = "https://graph.facebook.com/v2.6/";
+        String path = app_id+"/accounts/test-users";
+        return host + path;
+    }
+
+    private String getURLdeleteUser(String user_id) {
+        String host = "https://graph.facebook.com/v2.7/";
+        String path = user_id;
+        return host + path;
+    }
+
+    protected ArrayList<BasicNameValuePair> getParams() {
+
+        PARAMS = new ArrayList<BasicNameValuePair>();
+        PARAMS.add(new BasicNameValuePair("access_token", wikia_production_app_access_token));
+        return PARAMS;
+    }
+
+    private HttpResponse createTestUser(String app_id) throws IOException, URISyntaxException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00117)